ImageProcessingRequest_api

```diff
@@ -0,0 +1,86 @@
+v1.ImageProcessingRequest
+
+This is an object that describes request to the ChRIS system to be able to process an image into a desired format
+
+Fields:
+    - imageprocessingrequestid: an autogenerated id that corresponds to the submitted job
+    - patientid: The id of the patient
+    - jobtype: The type of image processing job you want to run
+    - jobstatus: The status of the job (e.g. 'Pending', 'Running', 'Failed', 'Success')
+
+
+
+
+create an ImageProcessingRequest
+
+POST /chrisapi/v1/imageprocessingrequests
+
+Parameters
+Type: BodyParameter
+Description: Contains the image
+Required: true
+Schema: Valid medical image format (e.g. .dcm, .nii, etc.)
+Default:
+
+Type: QueryParameter
+Name: patientid
+Description: The id of the patient
+Required: true
+Schema: string
+Default:
+
+Type: QueryParameter
+Name: jobtype
+Description: The type of image processing job you want to run
+Required: true
+Schema: string
+Default:
+	
+Responses
+HTTP Code: 200
+Description: success
+Schema: v1.ImageProcessingRequest
+
+Consumes
+    - application/dicom
+    - ...
+
+Produces
+    - application/json
+
+
+
+
+get an ImageProcessingRequest
+
+GET /chrisapi/v1/imageprocessingrequests/imageprocessingrequestid
+	
+Responses
+HTTP Code: 200
+Description: success
+Schema: v1.ImageProcessingRequest
+
+Produces
+    - application/json
+
+
+
+
+get an ImageProcessingRequest
+
+GET /chrisapi/v1/imageprocessingrequests/imageprocessingrequestid/processedoutput
+	
+Responses
+HTTP Code: 200
```

Comment:
I would expect a 404 if it's still running.  Meaning this api just looks for the results and returns them.  I wouldn't expect it to have to look up the original request and check the status.  Otherwise, you wouldn't really need status and you could just keep calling this api.  Are you thinking otherwise?